release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4207,4 +4207,23 @@ $ oc adm release info 4.12.39 --pullspecs
 [id="ocp-4-12-39-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-40"]
+=== RHSA-2023:5896 - {product-title} 4.12.40 bug fix and security update
+
+Issued: 2023-10-25
+
+{product-title} release 4.12.40, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:5896[RHSA-2023:5896] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5898[RHBA-2023:5898] advisory.
```

Comment:
https://access.redhat.com/errata/RHSA-2023:5896 is good but I get a 404 for https://access.redhat.com/errata/RHBA-2023:5898